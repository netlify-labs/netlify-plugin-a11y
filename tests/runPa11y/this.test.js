const path = require('path')
const filePath = path.relative(process.cwd(), path.join(__dirname, 'publishDir/index.html'))

// actual test
const pluginCore = require('../../src/pluginCore')
test('runPa11y works', async () => {
	const results = await pluginCore.runPa11y({
		htmlFilePaths: [filePath],
		build: { failBuild() {} },
	})
	expect(results).toMatchSnapshot()
})
