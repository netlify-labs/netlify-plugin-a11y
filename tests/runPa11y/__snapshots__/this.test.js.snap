// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`runPa11y works 1`] = `
Object {
  "issueCount": 1,
  "report": "
[4mResults for file: ./tests/runPa11y/publishDir/index.html[24m

[31m â€¢ Error:[39m Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
[90mâ”œâ”€â”€ WCAG2AA.Principle1.Guideline1_1.1_1_1.H37[39m
[90mâ”œâ”€â”€ html > body > img[39m
[90mâ””â”€â”€ <img src=\\"https://placekitten.com/200/300\\">[39m

[31m1 Error[39m
",
}
`;
